package.json

```diff
@@ -39,7 +39,9 @@
     "beautify": "beautify-rewrite lib/**.js test/*.js",
     "lint": "eslint lib test",
     "pretest": "npm run lint && npm run beautify-lint",
-    "test": "mocha --full-trace --check-leaks",
+    "test_old": "mocha --full-trace --check-leaks",
```

Comment:
Is this meant to be here? Is this meant to be here or were you thinking about reverting back.